nix/overlays/haskell-packages.nix

```diff
@@ -18,33 +21,14 @@ let
       # To get the sha256:
       #   nix-prefetch-url --unpack https://hackage.haskell.org/package/protolude-0.3.0/protolude-0.3.0.tar.gz
 
-      # TODO: We need to patch upstream for unbreaking hasql-dynamic-statements, hasql-implicits, ptr
       hasql-dynamic-statements =
-        self.haskell.lib.dontCheck (prev.callHackageDirect
-          {
-            pkg = "hasql-dynamic-statements";
-            ver = "0.3.1";
-            sha256 = "1zjv91xlfkyxwq6mhzj7rsfm4kjvs9ygkgbl6jbbg19jihcn2kiy";
-          }
-          { }
-        );
+        lib.dontCheck (lib.unmarkBroken prev.hasql-dynamic-statements);
```

Comment:
[unmarkBroken](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/haskell-modules/lib.nix#L172), nice escape hatch.

Reply:
Or `markUnbroken`, equivalently - all the choices you might want :-)